pkg/scheduler/framework/interface.go

```diff
@@ -131,7 +131,7 @@ func NewPodsToActivate() *PodsToActivate {
 }
 
 // Status indicates the result of running a plugin. It consists of a code, a
-// message, (optionally) an error and an plugin name it fails by.
+// message, (optionally) an error and a plugin name it fails by.
```

Comment:
should this have an oxford comma after `error`? ie `It consists of a code, a message, (optionally) an error, and a plugin name it fails by`

Reply:
@damemi I have added a comma, please help me to review it, thanks.